pages.de/common/pip-install.md

```diff
@@ -13,12 +13,12 @@
 
 - Installiere die Pakete aus einer Datei:
 
-`pip install -r {{requirements.txt}}`
+`pip install --requirement {{requirements.txt}}`
```

Comment:
```suggestion
`pip install -r {{requirements.txt}}`
```

Again, why the long option in this case? Nobody uses pip this way

Reply:
In my opinion, it makes sense to have the short option here, because almost every python project says in its README to run "pip install -r requirements.txt". I have never once seem someone use "--requirements".